tests/regression.sh

```diff
@@ -26,6 +26,15 @@ function fail()
     fi
 }
 
+function show_file()
```

Comment:
I chose to discard the cmp results because they are not really readable - they contain ANSI escape sequences and it's hard to tell what parts differ.  Maybe just print out their locations to help further debug, rather than cat them all?

Reply:
I don't think printing locations would be useful; at least in the pipeline Fedora uses the files won't be collected.

I do agree that the result of `cmp` in particular is not very helpful, though.   What would be a better option?

It seems tricky especially in case of ydiff, where ANSI codes are subject of the test, plus the output is already diff (sometimes diff of diff).  But as I mentioned in my comment to #125, I do have ideas to crack this.  I just did not want to "ambush" you with too big changes in one PR :)

How about I try to come up with better way of presenting the result?  It would be a bit more changes to this shell script but it might be worth it.   My idea is to normalize the outputs a little bit (basically just replace ANSI with escape codes, making it true utf8 plain text while preserving the structure); and then show simply the `diff -u` of the normalized versions of expected vs. actual output.  At that point the diff itself would be descriptive enough so that it would suffice to print just the diff and the stderr.  (Perhaps with some brief message to help the reader understand the output.)

Reply:
Sure, if you found a good solution. 

Keep in mind that when there is a regression, it's usually failing a lot of tests, so the result diff might be hard to read and often you will want to debug one case at a time - that's why I prefer printing out the paths of the temporary files for further looking. You are welcome to proceed your way if you do find a good balance.

Another thing is that such regressions are usually caught by unit tests first.